wn/util.py

```diff
@@ -153,3 +153,9 @@ def flash(self, message: str) -> None:
     def close(self) -> None:
         """Print a newline so the last printed bar remains on screen."""
         print(file=self.file)
+
+
+class GeneratorProgressInfo(NamedTuple):
+    """Progress Bar info for generator."""
+    total_items: int
+    progress_handler: ProgressHandler
```

Comment:
I'm not certain I see the need for this new class. What is it's purpose?

Reply:
The purpose is just holding the progress_handler and the total count to pass them downstream to the load function. Another way could just be changing again the signature of `lmf.load`, and pass the two arguments. But I don't think that will look clean.

Reply:
Actually my bad, I could have set the `ProgressHandler` before even calling `load`

Reply:
Ok yes, I see the issue: `lmf.load()` doesn't call `lmf.scan_lexicons()` itself so it doesn't have those counts. I think, for now, what you describe is sufficient: instantiating a ProgressHandler and passing the object (instead of the class) to `load()`. It breaks the current pattern, but the other options require too much work for this PR.

Reply:
Yes, but there is a solution, I think it might be cleaner than creating the class I was creating. I just set the progress before calling `lmf.load`. I think I have addressed this to maintain the pattern